src/python/ensembl/io/genomio/db_factory.py

```diff
@@ -19,48 +18,10 @@
 """
 
 import json
-from pathlib import Path
 from typing import Dict, List, Optional
 
-import argschema
-
 from ensembl.brc4.runnable.core_server import CoreServer
-
-
-class InputSchema(argschema.ArgSchema):
-    """Input arguments expected by this script."""
-
-    # Server parameters
-    host = argschema.fields.String(
-        metadata={"required": True, "description": "Host to the server with EnsEMBL databases"}
-    )
-    port = argschema.fields.Integer(metadata={"required": True, "description": "Port to use"})
-    host = argschema.fields.String(metadata={"required": True, "description": "Host to use"})
-    user = argschema.fields.String(metadata={"required": True, "description": "User to use"})
-    password = argschema.fields.String(metadata={"required": False, "description": "Password to use"})
-
-    # Filters
-    prefix = argschema.fields.String(
-        metadata={"required": False, "description": "Prefix to filter the databases"}
-    )
-    build = argschema.fields.String(
-        metadata={"required": False, "description": "Build to filter the databases"}
-    )
-    version = argschema.fields.String(
-        metadata={"required": False, "description": "EnsEMBL version to filter the databases"}
-    )
-    brc_mode = argschema.fields.Boolean(
-        metadata={
-            "required": False,
-            "description": "BRC4 mode: use organism_abbrev for species, component for division",
-        }
-    )
-    dbname_re = argschema.fields.String(
-        metadata={
-            "required": False,
-            "description": "regexp to match db name against",
-        }
-    )
+from ensembl.utils.argparse import ArgumentParser
```

Comment:
no dependency on `https://github.com/Ensembl/ensembl-py` added to `pyproject.toml`